ooniprobe/Utility/TestUtility.mm

```diff
@@ -144,6 +144,36 @@ + (void)downloadUrlsCallback:(NSData *)data
     successcb(urls);
 }
 
+//TODO unify this function and the above one
```

Comment:
Do you mean unify with `downloadUrlsCallback:`?

Reply:
yes, the two callbacks

Reply:
Can you maybe be explicit about the function names?